lib/components/artist/related_artists/related_artist.js

```diff
@@ -12,20 +12,24 @@ import ImageView from '../../opaque_image_view';
 class RelatedArtist extends React.Component {
   render() {
     const artist = this.props.artist;
+    const {height, width} = Dimensions.get('window');
+    const imageWidth = (width - 60)/2;
+    const imageAspectRatio = 1.5;
+    console.log(imageWidth, width);
     return (
-      <View style={{ width: 300 }}>
-        <ImageView style={{ height: 175, marginRight: 20 }}
-                   aspectRatio={1.7}
+      <View style={{ width: imageWidth }}>
+        <ImageView style={{ height: imageWidth/1.5 }}
+                   aspectRatio={1.5}
```

Comment:
If these need to be a fixed size, then you should be able to just specify the two dimensions and leave out the aspectRatio prop, saves a bit of calculations during the render pass.